libs/client/client.go

```diff
@@ -101,11 +111,13 @@ func (client *Client) Whitelist(ctx context.Context, veloOp vtxnbuild.Whitelist)
 	}, err
 }
 
+// Setup credit result struct contains success result from client.
 type SetupCreditResult struct {
 	HorizonResult  *horizon.TransactionSuccess
 	VeloNodeResult *cenGrpc.SetupCreditOpResponse
 }
 
+// Setup credit calling velo node to perform setup credit operation
```

Comment:
SetupCredit submits a VeloTx that contains SetupCreditOp to Velo Node and submits the result XDR to Horizon server.